app/templates/src/static/css/cf-enhancements.less

```diff
@@ -0,0 +1,10 @@
+/* ==========================================================================
+   Capital Framework
+   Enhancements
+   ========================================================================== */
+
+// This is your project's CF enhancements.
+//
+// Whenever building enhancements or changes to existing CF components, write them here
+// to work out the details before contributing them back to Capital Framework Project
```

Comment:
Not nitpicking, I entirely agree and would always rather do it now than later. Updated in fbbfbcb